cypress/integration/unit_tests/add_disaster_test.js

```diff
@@ -184,6 +184,32 @@ describe('Unit tests for add_disaster page', () => {
         })
         .then((success) => expect(success).to.be.true);
   });
+
+  it('deletes a disaster', () => {
+    cy.on('window:confirm', () => true);
+
+    const id = '2002-winter';
+    const states = ['DN, WF'];
+
+    cy.wrap(writeNewDisaster(id, states))
+        .then(() => {
```

Comment:
Nit: you can remove braces from this and below

Reply:
Done.

Reply:
Don't see it?